The build seems to be failing on travis but passing locally, we should look into this

See https://travis-ci.org/jembi/openhim-core-js/builds/67978019

Can reproduce locally by following the same steps as Travis (e.g. for build [1680](https://travis-ci.org/jembi/openhim-core-js/jobs/68150424)):
```
$ git clone --depth=50 --branch=master git://github.com/jembi/openhim-core-js.git jembi/openhim-core-js
$ cd jembi/openhim-core-js/
$ git checkout -qf 882e5db855320e7109a9d408a9561b79f9f683e0
$ npm install
$ grunt test
```

Results in
```
  362 passing (21s)
  5 failing

  1) API Metrics Tests openHIM Metrics Api testing *getGlobalLoadTimeMetrics() should fetch dashboard channel metrics based on the currently logged in user permissions:
     Uncaught AssertionError: expected [] to have property length of 2 (got 0)
      at Test.<anonymous> (test/integration/metricsAPITests.coffee:335:36)
      at _stream_readable.js:908:16

  2) API Metrics Tests openHIM Metrics Api testing *getGlobalStatusMetrics() should fetch global status metrics based on the currently logged in user permissions:
     Uncaught AssertionError: expected [] to have property length of 2 (got 0)
      at Test.<anonymous> (test/integration/metricsAPITests.coffee:351:36)
      at _stream_readable.js:908:16

  3) API Metrics Tests openHIM Metrics Api testing *getLoadTimeMetrics() should fetch channel metrics based by ID:
     Uncaught AssertionError: expected [] to have property length of 1 (got 0)
      at Test.<anonymous> (test/integration/metricsAPITests.coffee:370:36)
      at _stream_readable.js:908:16

  4) API Metrics Tests openHIM Metrics Api testing *getStatusMetrics() should fetch global status metrics based on the currently logged in user permissions:
     Uncaught TypeError: Cannot read property 'should' of undefined
      at Test.<anonymous> (test/integration/metricsAPITests.coffee:386:26)
      at _stream_readable.js:908:16

  5) Transaction Reports Reports should return a channel Report:
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
```

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "koa-route": "^2.4.0",
     "koa-statsd": "^0.0.2",
     "moment": "^2.8.4",
-    "mongoose": "^4.0.1",
+    "mongoose": "4.0.2",
     "nconf": "^0.7.1",
     "node-uuid": "^1.4.1",
     "nodemailer": "^1.3.2",

